pkgs/applications/window-managers/sway/contrib.nix

```diff
@@ -81,12 +88,17 @@ inactive-windows-transparency = python3Packages.buildPythonApplication rec {
   propagatedBuildInputs = [ python3Packages.i3ipc ];
 
   installPhase = ''
-    install -Dm 0755 $src/contrib/${lname}.py $out/bin/${lname}.py
+    install -Dm 0755 $src/${lname}.py $out/bin/${lname}.py
   '';
 
-  meta = sway-unwrapped.meta // {
```

Comment:
FYI: You could deduplicate this a bit if you want (homepage, license, platforms, and maintainers) but it isn't really relevant for just two packages.

Reply:
fair point, i did so
and i've set `meta.mainProgram` for the `inactive-windows-transparency.py`
(or is it preferred to remove the `.py` in this case?)